system-tests/projects/runner-e2e-specs/cypress/e2e/errors/events.cy.js

```diff
@@ -1,18 +1,18 @@
 describe('event handlers', { defaultCommandTimeout: 0 }, () => {
```

Comment:
Refactoring the error conditions here to use features available in both E2E and CT tests so it can be used in both